### PR TITLE
add script for migrations

### PR DESCRIPTION
**Pull request by jakazzy**
#### Pre-Submission Checklist

-   [x] Your pull request targets the `main` branch.
-   [x] Branch name is descriptive and follows the guidelines [here](https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices).
-   [ ] All tests pass . Use `git commit --amend` to amend any fixes.

#### Checklist:

-   [ ] All tests are passing.
-   [ ] Tested changes locally.
-   [ ] Improved documentation or readme where necessary.
-   [ ] Your commit messages format follows our commit message guideline [here](https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices#commits).

#### What does this PR do?
- [ ] Adds Migrations to the project
- [ ] Adds script for creating a new user
- [ ] Updates the README.md
- [ ] Validates envs

#### How should this be manually tested?
1. Clone the branch
2. npm install 
3.  run  `npm start`. This starts the server and applies the migrations

### CODE REVIEW

**Comment by jocrah** [comment]
@jakazzy, as this is a WIP, could you kindly give an idea of what is left of this?

**Comment by jakazzy** [comment]
@jocrah 
- [ ] refactor the script for seeding the database for users.

- [ ] I am also considering adding a script to seed the database with posts, comments and tags

File: .env.sample [hunk 1]
```diff
@@ -0,0 +1,6 @@
+DB_USER=
```
Comment (jocrah) [inline .env.sample:1]: Let's kindly add values to this, just some sample values to serve as a guide. 

I'm thinking in most cases the likely ones to be changed will be the user and password,

PS: Usually, its just the `env.sample` that is copied/duplicated to serve as the .env so might help

Comment (jakazzy) [reply .env.sample:1]: ok noted. Thanks

**Review by jocrah — APPROVED** [review]

**Pull request merged**